sphinx/analysis.rst

```diff
@@ -0,0 +1,36 @@
+.. methods:
+
+Analysis Methods
+################
+
+Analysis Methods work on PlantData objects to produce high level analysis products, such as the long term AEP.
+These methods rely on the more generic toolkit modules, chaining them together to create reproducible analysis processes.
```

Comment:
It would be helpful to briefly list and summarize all the analysis methods at the top and cite our papers where they have been used. I can work on this.

Also, should "toolkit modules" be changed to "utils modules" above to reflect v3?